src/modules/utils/marketplace.ts

```diff
@@ -34,3 +42,17 @@ export const errorCodeHelper = (err: string) => {
     message: errorCode ? errorMsg : err,
   };
 };
+
+export const getAuctionHouseInfo = (listing: AhListingMultiMarketplace): AUCTION_HOUSE_INFO => {
+  return (
+    AUCTION_HOUSE_ADDRESSES.find(
+      (ah) => ah.address === listing?.auctionHouse?.address.toString()
+    ) ||
+    MARKETPLACE_PROGRAMS.find((ah) => ah.address === listing?.marketplaceProgramAddress) || {
+      name: 'Unknown Marketplace',
+      address: null,
+      logo: '/images/listings/unknown.svg',
+      link: '',
+    }
+  );
+};
```

Comment:
This is possible a good option for a graphql client side field resolve. A good spot to grab the info until its moved to the API.

https://www.apollographql.com/docs/react/local-state/managing-state-with-field-policies

Reply:
Also instead of modules/utils/marketplace this should be in a modules/marketplace. Lets try not to have a utils module it just means we haven't come up with a good grouping name for the set of functions.